test-utils/src/main/java/io/littlehorse/test/internal/ExternalTestBootstrapper.java

```diff
@@ -18,10 +17,7 @@ public ExternalTestBootstrapper() {
             throw new IllegalStateException(String.format("Configuration file %s doesn't exist", LH_CONFIG_FILE));
         }
         workerConfig = new LHConfig(configPath.toString());
-        lhClient = workerConfig
-                .getBlockingStub()
-                .withCallCredentials(new StandaloneTestBootstrapper.MockCallCredentials(
-                        new TenantIdModel(workerConfig.getTenantId())));
+        lhClient = workerConfig.getBlockingStub();
```

Comment:
Setting the call credentials is the responsibility of the `LHConfig`, so we don't need to do it here.